lib/utils/index.js

```diff
@@ -398,5 +440,69 @@ module.exports = {
         cb(node.arguments.slice(-1)[0])
       }
     }
+  },
+
+  iterateProperties (node, groups, cb) {
```

Comment:
There is the reason you avoid generator functions by callback style?
I like `for-of`.